src/app/navbar/navbar.component.ts

```diff
@@ -7,6 +7,8 @@ import { Component, OnInit } from '@angular/core';
 })
 export class NavbarComponent implements OnInit {
 
+  isNavbarCollapsed=true;
+
   constructor() { }
 
   ngOnInit(): void {
```

Comment:
I think I left this in here by accident when I was trying other stuff. I'll take it out in my next commit. (don't want to take it out now in case it breaks something :laugh:)